packages/bsky/src/services/feed/index.ts

```diff
@@ -70,6 +70,7 @@ export class FeedService {
   async getActorViews(
     dids: string[],
     viewer: string | null,
+    skipLabels?: boolean, // @NOTE used by composeFeed() to batch label hydration
```

Comment:
for non-obvious boolean vals like this, I kinda like wrapping it in an object so it's clear from the caller what you're asking for like
```ts
getActorViews(dids, requester, { skipLabels: true })
```

totally unnecessary, but could be a good convention to establish. I started doing it in some of our services etc. thoughts?

also in cases where you have a bunch of string parameters that could get mis-arranged or something

Reply:
I'm 100% down with that convention, really just reached for this because I thought it was more standard in our codebase 😆 I will tidy that up!

Reply:
oops I forgot about this thread & merged the PR lol 🙃

Reply:
put it in the quality of life queue lol

Reply:
Hahaha sounds good!